fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -1551,7 +1552,8 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 				traceBatch = { TraceBatch{} };
 				traceBatch.get().addEvent("GetValueDebug", a.debugID.get().first(), "Reader.Before");
 			}
-			if (SERVER_KNOBS->ROCKSDB_SET_READ_TIMEOUT && readBeginTime - a.startTime > readValueTimeout) {
```

Comment:
Could we have a counter / histogram to track the number of reads taking more than 5sec?

Reply:
we have a log event printed... so we can use a splunk query to calculate the counts if needed